src/components/WwwFrame/TheHeader.vue

```diff
@@ -576,7 +575,11 @@ export default {
 			return route;
 		},
 		loadLendInfo() {
-			this.$refs.lendMenu.onLoad();
+			if (this.mgHighlightInNavVersion === 'shown') {
```

Comment:
This starts loading the exp "Borrower" menu when the first lend dropdown is shown, which leads to nice UI behavior since it starts loading the categories before the user has decided to hover over "Find a Borrower"